api/src/services/connected-user-log.js

```diff
@@ -0,0 +1,47 @@
+const moment = require('moment');
+const db = require('../db');
+
+const LOG_TYPE = 'connected-user-';
+const TIME_INTERVAL = 30 * 60 * 1000; // 30 minutes
```

Comment:
Setting this so that we are not updating the log too often especially because dbinfo is called a couple of times when syncing.

Reply:
Thanks for this comment. Can you rename `TIME_INTERVAL` to match that concept?